hack/make-bundle-vars.mk

```
@@ -36,6 +36,7 @@ SKIP_RANGE ?=
 # Image URL to use all building/pushing image targets
 IMAGE_REGISTRY ?= quay.io
 REGISTRY_NAMESPACE ?= ocs-dev
+CSI_ADDONS_IMAGE_REGISTRY ?= quay.io
```

```suggestion
CSI_ADDONS_IMAGE_REGISTRY ?= $(IMAGE_REGISTRY)
```

Probably useful to only have to set it once but be able to override it.